x-pack/plugins/ml/public/timeseriesexplorer/components/delete_annotation_modal/index.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {
+  EuiConfirmModal,
+  EuiOverlayMask,
+  EUI_MODAL_CONFIRM_BUTTON,
+} from '@elastic/eui';
+
+export function DeleteAnnotationModal({
+  cancelAction,
+  deleteAction,
+  isVisible
+}) {
+  return (
+    <React.Fragment>
+      {isVisible === true &&
+        <EuiOverlayMask>
+          <EuiConfirmModal
+            title="Delete this annotation?"
+            onCancel={cancelAction}
+            onConfirm={() => deleteAction(this)}
```

Comment:
I wouldn't think you'd need to pass `this` here - any reason you are? I think you should be able to just add `deleteAction` like `cancelAction` above. Might be missing some context though 🤔

Reply:
oh thanks for spotting this that's a leftover after some testing!

Reply:
Removed in cbad20e421.